packages/flutter_tools/lib/src/build_system/build_system.dart

```diff
@@ -490,9 +540,18 @@ class BuildSystem {
 
 /// An active instance of a build.
 class _BuildInstance {
-  _BuildInstance(this.environment, this.fileCache, this.buildSystemConfig)
-    : resourcePool = Pool(buildSystemConfig.resourcePoolSize ?? globals.platform?.numberOfProcessors ?? 1);
-
+  _BuildInstance(
+    this.environment,
+    this.fileCache,
+    this.buildSystemConfig,
+    this._logger,
```

Comment:
Everywhere else we've been using named parameters. Why not here, too?

Reply:
Done. I made them public, since the overall class is private it doesn't matter